### ISSUE TITLE
Multiple bad import (not absolute nor relative)

### ISSUE THREAD

**Issue by m-lamour** [wl-online-payments-direct/sdk-python3#1]
An import is wrong in the files :
* ingenico/direct/sdk/defaultimpl/default_connection.py
* ingenico/direct/sdk/api_resource.py
* ingenico/direct/sdk/communicator.py
* ingenico/direct/sdk/api_exception.py
* ingenico/direct/sdk/webhooks/web_hooks_helper_builder.py
* ingenico/direct/sdk/webhooks/web_hooks_helper.py

The import must be absolute or fully relative not a mix of both.

```
Traceback (most recent call last):
[...]
  File "[...].py", line 3, in <module>
    from ingenico.direct.sdk.factory import Factory
  File "[...]/.venv/lib/python3.7/site-packages/ingenico/direct/sdk/factory.py", line 4, in <module>
    from ingenico.direct.sdk.defaultimpl.default_connection import DefaultConnection
  File "[...]/.venv/lib/python3.7/site-packages/ingenico/direct/sdk/defaultimpl/default_connection.py", line 10, in <module>
    from direct.sdk.proxy_configuration import ProxyConfiguration
ModuleNotFoundError: No module named 'direct'
```

**Comment by worldline-direct-support-team** [comment]
Hi @m-lamour,

Thanks for the information! Relative imports have been replaced with absolute imports in release 1.3.1 and 1.4.0. Could you verify that the change works for you?

Kind regards,
Ingenico Direct support team

**Comment by m-lamour** [comment]
Hi,
Yes the issue is resolved since the 1.3.1.

Thanks
